airbyte-integrations/connectors/source-stripe/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: e094cb9a-26de-4645-8761-65c0c425d1de
-  dockerImageTag: 5.4.9
```

Comment:
Is this a patch version upgrade? Or should it be something else?

I just took that from the original PR, but let me know if this should be some other kind of version change instead.

Reply:
Yes, it's considered as `patch`.